docs/blocks/blocks-16.md

```diff
@@ -819,6 +819,12 @@ bedrock_wiki:
 
 .. Will, in-game, yield the result: `tile.Barrel of Foxes.name`. Using language files is strongly recommended.
 
+-   You can define a specific block property in the `/fill` and `/setblock` commands where the data value would be, just as shown in the following example;
```

Comment:
I don't think this is necessary as you can just add a note that custom block properties can be set via fill and setblock just like normal blocks